### DISCUSSION TITLE
[draft-v1] Figure 3: Faculty vs students

### DISCUSSION THREAD

**Discussion by pllim** [astropy/astropy-v5.0-paper#50] [Figure comments] [answered]
Students have obviously decreased over time, though it in itself is a small-number statistic, while faculty remains stagnant.

The students who stayed on are now in either the faculty or software engineer bin. This also means we are not recruiting a lot of new students. Even if we do, do we have the capacity to train them in addition to everything else we need to do to keep the project afloat?

Given how to long it takes to get to the faculty position in academia and how long one tends to stay there (especially when tenured), it is not surprising to see the faculty bins being flat across time.

**Comment by pllim** [comment]
cross-post: https://github.com/astropy/astropy-v5.0-paper/discussions/49#discussioncomment-2390053

**Comment by hamogu** [comment]
Post-Docs are already counted as "research staff" for the purposes of this table. It's not hard to become a Post-Doc.

**Reply by adrn** [reply]
I'm not sure what you mean by "It's not hard to become a Post-Doc" here?

**Comment by adrn** [comment, marked as answer]
Thanks for raising these points! I'm going to mark this as answered unless you would like to incorporate some more discussion into the text?
